README.md

```diff
@@ -249,29 +249,26 @@ Consumers of the EOSIO DID Method implementation are RECOMMENDED to extend the D
 
 At least one service SHOULD exist on a DID Document of LinkedDomains type. This can be used to resolve the DID and connect to the EOSIO chain through a supported API.
 
-Registered EOSIO chain names should add at least one servic in the [EOSIO DID chain method json registry](https://github.com/Gimly-Blockchain/eosio-did-spec/blob/master/eosio-did-chain-registry.json).
+Registered EOSIO chain names should add at least one service in the [EOSIO DID chain method json registry](https://github.com/Gimly-Blockchain/eosio-did-resolver/blob/master/eosio-did-chain-registry.json).
 
-**QUESTION: should we and how can we specify the EOSIO chain protocol version support of the DID? e.g. is the chain 2.0 or 2.4 or 1.8 compatible...**
-
-### 5.4.1 Service Types
-
-Multiple different APIs exist within the EOSIO ecosystem.
-1. [Nodeos HTTP API](https://developers.eos.io/manuals/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index) - default provided by Block One, author of EOSIO
-2. [Dfuse](https://dfuse.io) - provides websocket connection and history search features
-3. [Hyperion](https://hyperion.docs.eosrio.io) - provides history search features
-4. [EOSIO Light API](https://github.com/cc32d9/eosio_light_api) - provides history search features
-
-A [service type](https://w3c.github.io/did-spec-registries/#service-types) MUST be provided to describe the type for services related to access and transaction of an EOSIO blockchain.
-
-| API Name | Service type |
-| ------------- |-------------| 
-| Nodeos| EosioNodeos |
-| Dfuse Http Rest API | EosioDfuseRest |
-| Dfuse Websocket API | EosioDfuseWebsocket |
-| Hyperion | EosioHyperion |
-| EOSIO Light API | EosioLightAPI |
+```json
+{
+    [...],
```

Comment:
this is not valid json, Github does not render it well. see https://github.com/Gimly-Blockchain/eosio-did-spec/blob/d2770e658e8eb6fdeeaf0877b6c2f84447616d01/README.md#54-services
 please fix